index.js

```diff
@@ -20,15 +20,24 @@ const getStream = (config) => {
       stream = { path: config.streamPath || './logs/app.log' };
       break;
     case 'SYSLOG':
+      const syslogStream = bsyslog.createBunyanStream({
+        name: config.streamName,
+        host: config.streamHost,
+        port: parseInt(config.streamPort, 10),
+        facility: bsyslog.facility.local0,
+        type: config.streamProto
+      });
+      if (!config.streamProto // default type is udp, so undefined shall also be handled
```

Comment:
it's a bit verbose in a single `if` clause, can you put those checks into a function like `isProtoUDP`